Automattic/wp-calypso
# Emails: Hide email comparison for unsupported product monthly billing

#### Changes proposed in this Pull Request

Adds initial code to hide Google Card for monthly billing sets annually billing period as default.

#### Testing instructions

1. Go to a site with domains without email subscription
2. Click on 'Add email' for the desired site
3. Once you are in the comparison page, add at the end the following flag: `?flags=emails/new-email-comparison`
4. Check that annually is selected as first choice
5. Check that mobile support works
6. Check that monthly Google Workspace is not supported (not displayed)

**Remarks**: it is only visible through the path /email/{domain}/purchase/{site}?flags=emails/new-email-comparison

![image](https://user-images.githubusercontent.com/5689927/146901478-0f813eec-54ec-4c4d-bb3a-fa479e5b17c1.png)

Related to {1200182182542585-as-1201497551531376}
